### PR TITLE
[pod-gateway] Support for copying DNS search and options fields

### PR DESCRIPTION
**Pull request by angelnu**
**Description of the change**

When the `DNSPolicy` is set to `None`, K8S does not set search and options. This release sets these values from the webhook.

**Benefits**

partial (without svc.cluster.local) name resolutions working

**Possible drawbacks**

NA

**Applicable issues**

NA

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/stable/pod-gateway/Chart.yaml [hunk 1]
```diff
@@ -2,14 +2,15 @@ apiVersion: v2
 appVersion: 1.2.3
```
Comment (bjw-s) [inline charts/stable/pod-gateway/Chart.yaml:2]:
```suggestion
 appVersion: 1.2.5
```

File: charts/stable/pod-gateway/README.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
 # pod-gateway
 
-![Version: 2.0.0](https://img.shields.io/badge/Version-2.0.0-informational?style=flat-square) ![AppVersion: 1.2.3](https://img.shields.io/badge/AppVersion-1.2.3-informational?style=flat-square)
+![Version: 2.1.0](https://img.shields.io/badge/Version-2.1.0-informational?style=flat-square) ![AppVersion: 1.2.3](https://img.shields.io/badge/AppVersion-1.2.3-informational?style=flat-square)
```
Comment (bjw-s) [inline charts/stable/pod-gateway/README.md:3]: The appVersion change will require a new README run as well

**Review by bjw-s — CHANGES_REQUESTED** [review]

File: charts/stable/pod-gateway/Chart.yaml (hunk 1 above)
Comment (angelnu) [reply charts/stable/pod-gateway/Chart.yaml:2]: done

File: charts/stable/pod-gateway/README.md (hunk 2 above)
Comment (angelnu) [reply charts/stable/pod-gateway/README.md]: done

**Review by bjw-s — APPROVED** [review]

**Pull request merged**